Feedback UI

Hard-Coder05 opened the pull request:
## Fixes #59 

### Description


**Type of Change:** (Delete irrelevant options)
- Code
- User Interface

**Code/Quality Assurance Only** (Delete irrelevant options)

- New feature (non-breaking change which adds functionality pre-approved by mentors)


### How Has This Been Tested?

tested locally


### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] I have added one/more required dependencies
- [x] I have formatted the code using dart format (dartfmt)


**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Hard-Coder05 committed:
Feedback UI Initial

Hard-Coder05 committed:
Completed only icons left

Hard-Coder05 committed:
Completed only icons left

Hard-Coder05 committed:
Completed UI

Hard-Coder05 committed:
Completed UI

Hard-Coder05 commented:
@werainkhatri ready fo review
<img width="250" src="https://user-images.githubusercontent.com/54059881/108074867-ae30cc80-708f-11eb-9ba2-927f0ab787b0.jpg">

werainkhatri commented on lib/screens/home/tabs/contact_us/contact_us.dart:
```diff
@@ -27,23 +33,312 @@ class ContactUsScreen extends StatelessWidget {
+                    padding: EdgeInsets.only(top: 20 * heightFactor),
+                    child: Row(
+                      children: [
+                        RichText(
```
this can be just Text. Use rich text when you have a line(s) with different parts having different font properties.

werainkhatri commented on lib/screens/home/tabs/contact_us/contact_us.dart:
```diff
@@ -27,23 +33,312 @@ class ContactUsScreen extends StatelessWidget {
+    }
   }
 
+  String _validator(String email) => email.isEmpty ? "" : null;
```
1. `email`?
2. No need to create a function. place the body directly in the widget as argument.

werainkhatri reviewed (changes requested):
1. Text of location, number and email should be bold
2. top Icons seem to be of different sizes
3. location, number and email, when clicked, should open specific links
4. Twitter link is incorrect (Wrong account)
5. Selected bottom navigation button should have the colorful background (refer design)
6. Big keyboards hide the feedback message edit text, do something to fix that (test it in a samsung phone)
7. Label in the feedback edittext is supposed to be `message`, not `Feedback`
8. `Submit` text is supposed to be bold
9. Reduce font size of `Feedback` text
10. spread the elements on the screen vertically, there is empty space in the bottom
11. Space b/w feedback icon and edittext is more than shown in design
12. Feedback Icon and edittext don't seem to be aligned
13. Increase social icons' size

werainkhatri commented:
@Hard-Coder05 Update?

Hard-Coder05 committed:
Necessary changes done as asked in the review

Hard-Coder05 committed:
Necessary changes done as asked in the review

Hard-Coder05 committed:
merging done

Hard-Coder05 commented:
<img width="250" src="https://user-images.githubusercontent.com/54059881/108654256-4d215280-74ee-11eb-86db-4ca2225f64b2.jpg">

Hard-Coder05 commented:
> 1. Text of location, number and email should be bold
> 2. top Icons seem to be of different sizes
> 3. location, number and email, when clicked, should open specific links
> 4. Twitter link is incorrect (Wrong account)
> 5. Selected bottom navigation button should have the colorful background (refer design)
> 6. Big keyboards hide the feedback message edit text, do something to fix that (test it in a samsung phone)
> 7. Label in the feedback edittext is supposed to be `message`, not `Feedback`
> 8. `Submit` text is supposed to be bold
> 9. Reduce font size of `Feedback` text
> 10. spread the elements on the screen vertically, there is empty space in the bottom
> 11. Space b/w feedback icon and edittext is more than shown in design
> 12. Feedback Icon and edittext don't seem to be aligned
> 13. Increase social icons' size

1. done
2. nope I have made them of the same size
3. done
4. fixed
5. added
6. working on it
7. done
8. done
9. done
10. i guess it's looking cool now
11. done
12. done
13. done

werainkhatri commented:
3 -> change it so that If I click on the text, it redirects instead of icon.
6 -> update on this?
others -> Good work!

Hard-Coder05 committed:
changes done

Hard-Coder05 committed:
completed

Hard-Coder05 commented:
@werainkhatri done from my side

Hard-Coder05 committed:
Merging

Hard-Coder05 committed:
Completed

werainkhatri reviewed (approved):
Good work!
